tests/drivers/counter/counter_basic_api/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -1,23 +1,34 @@
 &timer0 {
+	status = "okay";
 	prescaler = <4>;
 };
 
 &timer1 {
+	status = "okay";
 	prescaler = <4>;
 };
 
 &timer2 {
+	status = "okay";
 	prescaler = <4>;
 };
 
 &timer3 {
+	status = "okay";
 	prescaler = <4>;
 };
 
 &timer4 {
+	status = "okay";
 	prescaler = <4>;
 };
 
 &rtc0 {
+	status = "okay";
+	ppi-wrap;
+};
+
+&rtc2 {
+	status = "okay";
 	ppi-wrap;
```

Comment:
Previously one instance was tested with `ppi-wrap` and one without. Shouldn't we keep this scenario?